elpy.el

```diff
@@ -1130,9 +1130,11 @@ PyPI, or nil if that's VERSION."
                         ""))
 
          (command (cond
-                   ((executable-find "pip")
-                    (format "pip install %s%s%s"
-                            user-option upgrade-option python-package))
+                    ((= (call-process elpy-rpc-python-command
+                          nil nil nil "-m" "pip" "--help") 0)
+                     (concat elpy-rpc-python-command
+                       (format " -m pip install %s%s%s"
+                               user-option upgrade-option python-package)))
```

Comment:
This should be `(format "%s -m pip install %s%s%s" elpy-rpc-python-command user-option ...)`